main_routers/music_router.py

```
@@ -65,6 +68,150 @@ def _patch_pyncm_async() -> None:
+                content_length = resp.headers.get('Content-Length')
+                declared_size = 0
+                if content_length:
+                    try:
+                        declared_size = int(content_length)
+                        if declared_size > MAX_MUSIC_SIZE:
+                            logger.warning(f"[Music Proxy] 音乐文件过大: {declared_size}")
+                            return JSONResponse(content={"success": False, "error": "音乐文件超过大小限制 (50MB)"}, status_code=413)
+                    except (ValueError, TypeError):
+                        pass
+
+                # 大文件（≥10MB）流式传输
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)